### ISSUE TITLE
Module not found: Can't resolve '@dqbd/tiktoken'  

### ISSUE THREAD

**Issue by ameywtf** [dqbd/tiktoken#18]
I added the package using yarn add but this error won't go away, tried using npm but still persists 

```
Module not found: Can't resolve '@dqbd/tiktoken'

https://nextjs.org/docs/messages/module-not-found

Import trace for requested module:
./pages/api/chatgpt.js
./components/form.js
./pages/index.js
```

![image](https://user-images.githubusercontent.com/96624879/224883430-91b17b25-e7f4-4ab6-a299-b766076462fd.png)

**Comment by dqbd** [comment]
Hello @ameywtf, please try the newest latest version (`1.0.0`), which should resolve the bundling issues.

**Comment by HaleyLeoZhang** [comment]
> 

I had a similar problem  
![image](https://user-images.githubusercontent.com/18374677/226123630-1de6d839-bdd2-44ee-9320-d8c42f7e6039.png)  

March 3,2023, was the last time I compiled successfully

**Comment by dqbd** [comment]
The issue should be fully resolved with 1.0.4.
